portal/server/api.js

```diff
@@ -120,7 +133,7 @@ const updateDealName = async (id, newName, token) => {
 const updateDealStatus = async (statusUpdate, token) => {
   const response = await axios({
     method: 'put',
-    url: `${urlRoot}/v1/deals/${statusUpdate._id}/status`, // eslint-disable-line no-underscore-dangle
+    url: `${urlRoot}/v1/deals/${statusUpdate._id}/status`,
```

Comment:
Is eslint passing without the es-lint-disable-no-underscrore-dangle? has it been globally configured?

Reply:
yeah it's passing without, not made any changes to configuration. Running trough all tests with no problem

Reply:
I've just checked and I added it to global config a couple of months ago :man_facepalming: